File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.java
import java.util.*;
public class Main {
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		int N = s.nextInt();
		int K = s.nextInt();
		int num = N - K;
		
		int answer = method(N) / (method(K) * method(num));
		
		System.out.println(answer);
	}
	
	public static int method(int number) {
		int mul = 1;
		for(int i = number; i > 0; i--) {
			mul *= i;
		}
		return mul;
	}
}

File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.kt
fun main() {
    val (N, K) = readln().split(" ").map { it.toInt() }

    println(method(N) / (method(K) * method(N - K)))
}

fun method(number: Int): Int {
    var mul = 1
    for(i in number downTo 1) {
        mul *= i
    }
    return mul
}